x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -465,7 +490,7 @@ export const RulesList = ({
     setShowErrors((prevValue) => {
       if (!prevValue) {
         const rulesToExpand = rulesState.data.reduce((acc, ruleItem) => {
-          if (ruleItem.executionStatus.status === 'error') {
+          if (ruleItem.lastRun?.outcome === RuleLastRunOutcomeValues[2]) {
```

Comment:
Should we check the feature flag and have two conditions instead of one or just a `OR`